pyprof/nvtx/nvmarker.py

```diff
@@ -273,31 +279,39 @@ def add_wrapper(mod, fn_name):
                                        ) and (type(mod) is not torch.jit.TopLevelTracedModule)
 
     def wrapper_func(*args, **kwargs):
+        global wrappers_enabled
 
-        # Push trace marker
-        nvtx.range_push(traceMarker(fn_name))
+        if (wrappers_enabled):
```

Comment:
no parentheses needed

Reply:
There are plenty of cases of using parentheses for if/elif all over pyprof.  Do you really need me to remove these?

Reply:
we have plenty of instances where this happens in dlprof.  While there are other instances that could be cleaned up, we try to keep the code as clean as possible when we catch something.

Reply:
Where what happens in dlprof?  David has asked for a style where we don't use parenthesis when we return values. That is it. I disagree with the style but I go along with it because that is how the rest of the project is styled.

The style of this project has over 100 instances of using parenthesis in IF statements. Removing parenthesis doesn't make it "clean".